internal/service/wafv2/schemas.go

```diff
@@ -1159,7 +1159,7 @@ func rateBasedStatementSchema(level int) *schema.Schema {
 				"limit": {
 					Type:         schema.TypeInt,
 					Required:     true,
-					ValidateFunc: validation.IntBetween(100, 2000000000),
+					ValidateFunc: validation.IntBetween(10, 2000000000),
```

Comment:
I would recommend removing the validation all together to prevent this from happening in the future.
The API will validate the input itself and return an error if an invalid integer is provided.

In my opinion no need to maintain validation on both sides.

Reply:
@DanielRieske I understand (I was quite pissed it blocked me), but I also find it useful to have a plan-time warnings about stacks. 

If the PR approval and release process could be expedited for this kind of PRs, then it's not such a big trouble to have these checks hardcoded.